utils/tests/db.go

```diff
@@ -17,21 +17,24 @@ package tests
 
 import (
 	"database/sql"
+	"sync"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
+var testMutex sync.Mutex
+
 // waitForFixtures waits up to 30 seconds to database fixtures (test_db) to be loaded.
 func waitForFixtures(tb testing.TB, db *sql.DB) {
 	var id int
 	var err error
 	for i := 0; i < 30; i++ {
+		testMutex.Lock()
```

Comment:
@percona-csalguero I am trying to fix build, because since Andrii created this PR build were always failed. We have data racing in tests and it happening on more places.I will mark it as draft once more until build will be fixed.